network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2021/fragments/accordion_section_3.html

```diff
@@ -29,12 +29,11 @@ <h2 class="heading-2 mb-3">{% trans "We found that the rate of regrets is 60% hi
 
 <div class="row flex-column-reverse flex-lg-row">
   <div class="col-12 col-lg-5">
-    {% trans "36" as 36_percent %}
     {% trans "of non-english videos were pandemic-related" as of_non_english_videos %}
-    {% trans "14" as 14_percent %}
     {% trans "of english videos were pandemic-related" as of_english_videos %}
-    {% include "../../youtube-regrets-shared/standout-stat.html" with number=36_percent label=of_non_english_videos id="non-english-videos-count-up" classes="mb-5" suffix='%' %}
-    {% include "../../youtube-regrets-shared/standout-stat.html" with number=14_percent label=of_english_videos id="english-videos-count-up" suffix='%' %}
+    {% trans "%" as localized_percentage_suffix context "Added after a number" %}
+    {% include "../../youtube-regrets-shared/standout-stat.html" with number=36_percent label=of_non_english_videos id="non-english-videos-count-up" classes="mb-5" suffix=localized_percentage_suffix %}
+    {% include "../../youtube-regrets-shared/standout-stat.html" with number=14_percent label=of_english_videos id="english-videos-count-up" suffix=localized_percentage_suffix %}
   </div>
```

Comment:
Localizing the suffix instead of the value alone, as it’s clearer.
We’re localizing it as some locales will want to add a space